packages/ui/react-ui-editor/src/themes/default.ts

```diff
@@ -90,6 +90,10 @@ export const defaultTheme: ThemeStyles = {
     color: get(tokens, 'extend.semanticColors.description.dark', 'rgba(255,255,255,.2)'),
   },
 
+  '.cm-dropCursor': {
+    border: '1px solid red',
```

Comment:
red?

Reply:
suggest

Reply:
just thinking we should use the theme colours probably. I'd defer to @thure on colour choices here